src/functions.ts

```diff
@@ -468,13 +468,19 @@ export const collectBankAccountForPayment = async (
   clientSecret: string,
   params: PaymentMethod.CollectBankAccountParams
 ): Promise<CollectBankAccountForPaymentResult> => {
+  const subscription =
+    params.onEvent &&
+    eventEmitter.addListener('onFinancialConnectionsEvent', params.onEvent);
```

Comment:
we have an existing paradigm for this in this file: https://github.com/stripe/stripe-react-native/blob/f1400cc6ab0a9f1d266cfdca000d02e1d4d4659c/src/functions.ts#L357

I think it _should_ be fine with this setup overall, but i'd rather keep it consistent with the other subscriptions we have

Reply:
Gotcha, will take a stab at it!